internal/amt/commands.go

```diff
@@ -350,11 +351,12 @@ func (amt AMTCommand) GetLANInterfaceSettings(useWireless bool) (InterfaceSettin
 	}
 
 	settings := InterfaceSettings{
-		IPAddress:   "0.0.0.0",
-		IsEnabled:   result.Enabled == 1,
-		DHCPEnabled: result.DhcpEnabled == 1,
-		LinkStatus:  "down",
-		DHCPMode:    "passive",
+		AmtIPAddress: "0.0.0.0",
```

Comment:
```suggestion
		IPAddress: "0.0.0.0",
```

recommended update to avoid introducing breaking changes